mobile/Screen/ForgotPasswordScreen.js

```diff
@@ -65,18 +147,252 @@ function ForgotPasswordScreen({ navigation }) {
     return true;
   };
 
+  // function that allows users to repress the digits containers to focus
+  const handleDigitsPress = () => {
+    ref?.current?.focus();
+  };
+
+  // function that breaks down the input to digits
+  const digitInput = (index) => {
+    const digit = code[index] || ' ';
+
+    return (
+      <View key={index} style={(digit === ' ') ? styles.digitInputEmpty : styles.digitInput}>
+        <Text style={styles.digitTextInput}>{digit}</Text>
+      </View>
+    );
+  };
+
+  const checkValidEmail = () => {
+    if (!validateEmail(email)) {
+      Alert.alert('Please enter a valid email to proceed');
+    } else {
+      return true;
+    }
+    return false;
+  };
+
+  // Check account inputs are filled appropriately before moving to next screen
+  const checkAccountInputs = () => {
+    if (password === '') {
+      Alert.alert('Please enter a password to proceed');
+    } else if (password.length < 6) {
+      Alert.alert('Password must be longer than 5 characters');
+    } else if (password !== confirmPassword) {
+      Alert.alert('Password confirmation does not match password');
+    } else {
+      return true;
+    }
+    return false;
+  };
+
+  const handleSendCode = async () => {
+    try {
+      const res = await axios.patch(`${URL}/passwordReset/create`, { email });
+      console.log(res);
+    } catch (err) {
+      console.error(err);
+      return err;
```

Comment:
You need a return in the `try` block as well or remove the return `err`

Reply:
made it return a bool because some page-setting functionality depends on it